src/v/utils/tests/BUILD

```diff
@@ -505,3 +505,22 @@ redpanda_cc_bench(
         "@seastar//:benchmark",
     ],
 )
+
+redpanda_cc_btest(
```

Comment:
Can we please write new tests in gtest?

Reply:
Sorry I had written this utility >1 year ago and already had a bunch of tests written in boost (before we had gtest support).